.changeset/fuzzy-phones-rule.md

```diff
@@ -0,0 +1,5 @@
+---
+'@astrojs/partytown': minor
```

Comment:
Oh actually, misread this. This can be a patch, the upstream package only had fixes

Reply:
Oh yeah, makes sense. What should I do? :)